hermes-console-vue/src/composables/use-consumer-groups/useConsumerGroups.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { dummyConsumerGroups } from '@/dummy/consumerGroups';
+import { useConsumerGroups } from '@/composables/use-consumer-groups/useConsumerGroups';
+import { waitFor } from '@testing-library/vue';
+import axios from 'axios';
+import type { Mocked } from 'vitest';
+
+vitest.mock('axios');
+const mockedAxios = axios as Mocked<typeof axios>;
+
+describe('useConsumerGroups', () => {
+  it('should hit expected Hermes API endpoints', async () => {
```

Comment:
```suggestion
  it('should hit expected Hermes API endpoint', async () => {
```
yet another little nitpick 😌